### ISSUE TITLE
Style Nautilus 3.30 Pathbar

### ISSUE THREAD

**Issue by taciturasa** [ubuntu/yaru#793]
Know this isn't being rolled into Ubuntu yet, but I expect it will be at some point, alongside the desktop icons extension. 

![screenshot from 2018-09-08 02-00-29](https://user-images.githubusercontent.com/18127936/45250907-008c3180-b30b-11e8-9570-ef6530ea8ad9.png)
![screenshot from 2018-09-08 01-59-53](https://user-images.githubusercontent.com/18127936/45250908-008c3180-b30b-11e8-9e10-388aad66b908.png)

**Comment by Feichtmeier** [comment]
Did they introduce a new UI element for this which has a new style class or did they use an old style class?

**Comment by clobrano** [comment]
I don't think so. It's just a different pathbar shape

**Comment by taciturasa** [comment]
For reference, here is its intended behavior in adwaita
![screenshot from 2018-09-08 11-02-58](https://user-images.githubusercontent.com/18127936/45255617-ee37e500-b356-11e8-9ef6-4a8a57ed86d7.png)
![screenshot from 2018-09-08 11-02-48](https://user-images.githubusercontent.com/18127936/45255618-ee37e500-b356-11e8-9f0a-f4cc9c7235b2.png)
![screenshot from 2018-09-08 11-02-43](https://user-images.githubusercontent.com/18127936/45255619-eed07b80-b356-11e8-9a5b-561a9c547f88.png)

**Comment by clobrano** [comment]
> Also, that view button with the arrow could look better under Yaru.

this one?

![screenshot from 2018-09-08 17-36-53](https://user-images.githubusercontent.com/2883614/45255857-d92a7880-b38d-11e8-988a-832f9427936c.png)

**Comment by clobrano** [comment]
This is the "home" button

![screenshot from 2018-09-08 18-54-39](https://user-images.githubusercontent.com/2883614/45256593-d719e700-b398-11e8-90fa-7b2c928c73e1.png)

however we can work on it only after branching for 18.10

**Comment by taciturasa** [comment]
@clobrano indeed; that arrow and the button to the left of it are grouped in the original, but don't look grouped here.

**Comment by taciturasa** [comment]
Hey, small little update: https://gitlab.gnome.org/GNOME/nautilus/issues/661#note_333942 Due to the results of this issue, changes to work with High Contrast most likely means Yaru won't have to change much, if at all, to fix the issues present with the nautilus pathbar in versions past 3.30, or perhaps a 3.30 bugfix release. 😄

**Comment by Feichtmeier** [comment]
Closing this as we have another issue with more comments which is about the same topic but its on the first page of issues =)
Best regards and thanks for reporting

**Comment by Feichtmeier** [comment]
https://github.com/ubuntu/yaru/issues/914
